docs/src/design/chiplets/main.md

```diff
@@ -172,35 +191,53 @@ Lookup requests are sent to the chiplets bus by the following components:
 
+The chiplet bus can be initialized with reduced request $v_0, v_1, \ldots$ by computing their product $v_{init} = \prod_i v_i$, and enforcing the boundary constraint in the first row to ensure
+$b_{chip} = \frac{1}{v_{init}}$.
+Note that $v_{init}$ is a constant, and therefore does not contribute to the constraint degree.
+
+> $$
+b_{chip} \cdot v_{init} - 1 = 0 \text{ | degree} = 1
+$$
+
 ## Chiplets virtual table
 
-Some chiplets require the use of a [virtual table](../lookups/multiset.md#virtual-tables) to maintain and enforce the correctness of their internal state. Because the length of these virtual tables does not exceed the length of the chiplets themselves, a single virtual table called $vt_{chip}$ can be shared by all chiplets.
+_Note: over time, the use of this construction has evolved to the point where its name doesn't match the way it is used. This is documented in [issue #1779](https://github.com/0xMiden/miden-vm/issues/1779)._
 
-Currently, the chiplets virtual table combines two virtual tables:
-- the hash chiplet's [sibling table](./hasher.md#sibling-table-constraints)
-- the kernel ROM chiplet's [kernel procedure table](./kernel_rom.md#kernel-procedure-table-constraints)
+The [virtual table](../lookups/multiset.md#virtual-tables) bus $vt_{chip}$ is used by chiplets by several chiplets as a way to maintain and enforce the correctness of their internal states, and enable communication with each other.
 
-To combine these correctly, the [running product column](../lookups/multiset.md) for this table must be constrained not only at the beginning and the end of the trace, but also where the hash chiplet ends and where the kernel ROM chiplet begins. These positions can be identified using the chiplet selector columns.
+The hasher chiplet uses it as to store [sibling nodes](./hasher.md#sibling-table-constraints) when performing a Merkle tree update.
+In particular, it expects an empty bus at the start of this operation, and ensures that all entries it inserts are removed once the new tree is finalized.
+Consequently, the column representing this table must be equal to 1 at the boundaries of the hasher chiplet's trace, preventing communication with other chiplets.
 
-### Chiplets virtual table constraints
+Other chiplets use the table as an extension of the chiplet bus, since both multi-sets are merged in the last row of the overall trace.
+
+This enables chiplets to make bus requests to other chiplets, without affecting the degree of the chiplet bus.
+As currently implemented, a single constraint is required to include all requests made by the main trace and corresponding responses from the chiplets.
+The degree of this constraint is the maximum of both message types and is currently reached by the requests by the main trace,
+preventing chiplets from performing their own requests using the same bus.
```

Comment:
Would "...performing any requests using the same bus." be better, or is there something more we want to express?